tests/chainer_tests/test_variable.py

```diff
@@ -366,6 +366,67 @@ def test_addgrad_gpu_to_another_gpu(self):
             c = cp.full(3, 30, dtype=np.float32)
         self.check_addgrad(a, b, c)
 
+    def test_debug_print_cpu(self):
```

Comment:
Other test seems to have separate / duplicated methods for ``debug_cpu`` and ``debug_gpu``. Is there any example which you intend for?